content/pizza-instructions/contents.lr

```diff
@@ -6,27 +6,34 @@ body:
 
 ### Schedule the Order with Ian's
 
-* Order 6 pizzas from Ian's Pizza State Street location, specifically requesting that they be sliced into 12 slices per pie -- 608-257-9248, ext. 1
-* Keep the order simple & vegetarian-friendly:
-  * 2 Mac&Cheese pizzas,
-  * 2 "All The Veg" pizzas, and
-  * 2 "Smokey the Bandit" pizzas
-* Ask for plates & napkins for 40
-* Please include a tip to the kitchen/driver in your payment
-* Schedule delivery to the Central Library room 302 for 6:30pm on the night of the event
-* Give them Ed's name & number as the primary contact, and he'll take the call from the delivery driver, forward the receipt info to you, etc.
+* Get in touch with a MadPy organizer who will be present for the event (via [Slack](madpy.slack.com) is preferred) to confirm your sponsorship and get their name and phone number. They will be primary contact for the order and manage the delivery at the event.
```

Comment:
To keep consisent with the sidebar, let's point to slack.madpy.com, both here and lower on the page

```suggestion
* Get in touch with a MadPy organizer who will be present for the event (via [Slack](slack.madpy.com) is preferred) to confirm your sponsorship and get their name and phone number. They will be primary contact for the order and manage the delivery at the event.
```